### PR TITLE
LPS - 174666, LPS-173424, LPS-174493

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LPS-174666
https://issues.liferay.com/browse/LPS-173424
https://issues.liferay.com/browse/LPS-174493

This is a continuation of https://github.com/liferay-bpm-qa/liferay-portal/pull/395 with Picklist test and updates. 

Reasoning for the Picklist edits is because the paths were written in a way that would always default to the first picklist or item in the list regardless of the variable used. This wasn't an issue at first since all of the picklist tests either only use one picklist/item or the test uses the first one in the list. Instead of rewriting the paths, I found an existing macro that could be used and removed the now unused ones.

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:test:object

**Comment by epark18** [comment]
ci:test:object

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectUpgrade.testcase [hunk 1]
```diff
@@ -17,7 +17,7 @@ definition {
 	}
 
 	@ignore = "Test Stub"
-	@priority = "5"
+	@priority = 5
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectUpgrade.testcase:20]: Minor nitpick but prob better to include fixing typos like this in the original commit to keep the commit history clean.

**Review by timpak — APPROVED** [review]
Really great work with the upgrade tests! There's a minor nitpick I have but there's more value moving this forward. LGTM!

**Comment by timpak** [comment]
ci:forward

**Pull request closed without merging**